index.html

```diff
@@ -126,7 +126,7 @@ <h2>Tonic</h2>
                     </ul>
                 </div>
                 <div class="project-button pro-btn-one">
-                    <button>See Project</button>
+                    <button class="modal-mobile-see-pro">See Project</button>
```

Comment:
You have done a good job on this milestone👍👍 Just a quick reminder that it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) in this milestone that our project section is generated dynamically using the array of objects you have created in your script file😉😉😉 kindly go through the requirements and make the required change🤗🤗

<img width="763" alt="pop__instruction" src="https://user-images.githubusercontent.com/64643995/218076331-36d445cf-bd0d-4e3f-839a-e33f32faf078.png">